### ISSUE TITLE
Support raw POST/PUT data

### ISSUE THREAD

**Issue by treyhunner** [Hipo/Django-API-Playground#12]
I use an API that accepts raw PUT and POST data as input.  It would be helpful if a textarea field could be used to send raw JSON data in the request body.

**Comment by Timmmm** [comment]
Agreed. Also it would be nice if it supported sending the form data as multipart/form-data instead of JSON.
